grails-app/conf/Config.groovy

```diff
@@ -1197,6 +1197,8 @@ openboxes {
                         [label: "documents.label", defaultLabel: "Documents", href: "/${appName}/document/list"],
                         [label: "documentTypes.label", defaultLabel: "Document Types", href: "/${appName}/documentType/list"],
                         [label: "eventTypes.label", defaultLabel: "Event Types", href: "/${appName}/eventType/list"],
+                        [label: "glAccountType.label", defaultLabel: "GL Account Type", href: "/${appName}/glAccountType/list", requiredRoles: [RoleType.ROLE_ADMIN, RoleType.ROLE_SUPERUSER]],
```

Comment:
This isn't a big deal, but can you figure out a way to implement the requiredRoles using the lowest allowed role. I know this is implemented somewhere using getEffectiveRoles() or getHighestRole(). We basically just want to include the lowest common denominator here. It's possible that the taglib that is used to check if a user has a role in the megamenu does this same logic. And if that's the case just leave it as-is and we'll refactor later.

Reply:
I'll create a ticket for this